### PR TITLE
{bio}[system/system] FreeSurfer v7.3.2

### PR DESCRIPTION
**Pull request by maxim-masterov**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/5ae32d92a377bbe234e96c333967042a for a full test report.

File: easybuild/easyconfigs/f/FreeSurfer/FreeSurfer-7.3.2-centos8_x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+description = """FreeSurfer is a set of tools for analysis and visualization 
+of structural and functional brain imaging data.  FreeSurfer contains a fully 
+automatic structural imaging stream for processing cross sectional and 
+longitudinal data."""
+
+toolchain = SYSTEM
+
+source_urls = [
+    'https://surfer.nmr.mgh.harvard.edu/pub/dist/freesurfer/%(version)s/',
+    'https://surfer.nmr.mgh.harvard.edu/fswiki/MatlabRuntime?action=AttachFile&do=get&target='
```
Comment (verdurin) [inline easybuild/easyconfigs/f/FreeSurfer/FreeSurfer-7.3.2-centos8_x86_64.eb:15]: @maxim-masterov could you check this part, please? It failed to download in my first test build attempt.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3100.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/63e4ed81c1f6e780262789e87440689e for a full test report.

File: easybuild/easyconfigs/f/FreeSurfer/FreeSurfer-7.3.2-centos8_x86_64.eb (hunk 1 above)
Comment (maxim-masterov) [reply easybuild/easyconfigs/f/FreeSurfer/FreeSurfer-7.3.2-centos8_x86_64.eb:15]: @verdurin Not sure why it fails, I've just re-installed it from this pr (removed old sources prior to it) and it worked

Comment (boegel) [reply easybuild/easyconfigs/f/FreeSurfer/FreeSurfer-7.3.2-centos8_x86_64.eb:15]: Download works fine for me

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/f604a3c4188c710ede15e12262583ee5 for a full test report.

File: easybuild/easyconfigs/f/FreeSurfer/FreeSurfer-7.3.2-centos8_x86_64.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/f/FreeSurfer/FreeSurfer-7.3.2-centos8_x86_64.eb:15]: Yes, I retried and it was fine.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @maxim-masterov!

**Pull request merged**